### DISCUSSION TITLE
Can you get SSH access to another private repo within the same organization?

### DISCUSSION THREAD

**Discussion by meenie** [community/community#12711] [Codespaces] [answered]
Greetings,

Our private repo does a `yarn install` to install an app from another one of our private repos within the same organization (which is enabled for Codespaces as well) but I'm getting the following error:

```
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.
```

That was after I did
```
ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
```

Before the above it was doing:
```
Output:
Host key verification failed.
fatal: Could not read from remote repository.
```

Running the devcontainer.json file locally in VS Code without involving GitHub Codespaces works just fine.

**Comment by meenie** [comment, marked as answer]
Seems this is not possible. Which means you need to create an SSH key in the new container, start up the ssh agent, register the key, and then configure the key with GitHub.

**Comment by dlants** [comment]
Pretty disappointing that this doesn't work. I feel like this is a pretty common use case - a node project w/ yarn or npm pulling from a private repo via `git+ssh`.

**Reply by andyshinn** [reply]
I agree. Just was trying it out and the first issue I ran into was I couldn't add all the packages for my Python project because some are referenced as git repos. It is a non-starter if I cannot pull down private git repos.
